.github/workflows/azure-static-web-apps-gentle-beach-0eb2d061e.yml

```diff
@@ -3,16 +3,19 @@ name: Azure Static Web Apps CI/CD
 on:
   push:
     branches:
-      - main
+      - Assignment1
     paths:

```

Comment:
Is paths necessary?

Reply:
We only wanted to pull code from that specific branch we just forgot to change it

Reply:
The site should be up now if you are curious and want to take a look